README.md

```diff
@@ -13,6 +12,21 @@
 1. Start Migrator (start.sh/start.bat) and wait to finish.
 1. Navigate to Operate and check result.
 
+## Development Setup
```

Comment:
We can try to combine with the previous point `How to use the Migrator`. What do you think?